### DISCUSSION TITLE
Whitelist www.ecosia.org

### DISCUSSION THREAD

**Discussion by u3erprutser** [uBlockOrigin/uBlock-issues#3189] [General]
Ecosia want me to disable my ad-blocker. And I don't mind a few ads, so they can plant some trees.
But using ctrl+click on the big on off button, unfortunately, is not a permanent solution.
Do I perhaps need to configure a filter? If so, I could use some help. Because reading through the help pages, it's rather overwhelming.

**Comment by ghost** [comment]
Just the "click", not Ctrl+click.

**Reply by ghost** [reply]
Maybe the tooltip should say:

```
Click to disable uBlock₀ for this |whole| site.

Ctrl+click to disable uBlock₀ only on this |one| page.
```

**Reply by garry-ut99** [reply]
> gwarser : Maybe tooltip should say:


I don't think so, one shouldn't propose changes without even asking OP why he whitelisted a page instead of a site, given that the tooltip and the documentatioin is clear enought :  https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-large-power-button, most likely it's OP's fault he failed to understand the difference between a website and a webpage, in such a case he should educate: https://www.javatpoint.com/webpage-vs-website.

> u3erprutser : Because reading through the help pages, it's rather overwhelming.

It's not overwhelming, problem is on your side - given it's you who has problems reading a simple tooltip or understanding difference between a page and a site, we can assume it's also you who has problem reading/understaning the repository readme file and the documentation.

uBO documentation is very good, written in plain english, not in japanese, it has table of contents and search box as well, as proof - what you're asking can be quickly found and is already greatly explained:  https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-large-power-button, purpose of such a good documentation is to cause users like you to not spam issue tracker by invalid or duplicate issues and to not ask questions which are already well documented.

You also didn't read or didn't understand the repository readme which clearly says where to ask questions: 

> https://github.com/uBlockOrigin/uBlock-issues?tab=readme-ov-file#support-forum:
> For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).

Is that because the repository readme file is "overwhelming" too? No, it's small, simple and clear, either you have problems understanding even simple things like a readme file, a tooltip, or everything is "overwhelming" for you.

What is overwhelming is users like you, the developers and free volunters spent many years to create such a good and detailed repository readme file and documentation, and now you can't spent several minutes to read or to understand it properly, you should respect the readme and the documentation instead of spamming.

I also assume you're not a person with disability, because you didn't mention about anything like that, hence my reply is crafted for typical users who can read and understand documentation files without problems.

Also related/similiar invalid issues :
- https://github.com/gorhill/uBlock/issues/1151
- https://github.com/gorhill/uBlock/issues/2214
